From f873c32538f9b920c452354665e8957abaf3d63a Mon Sep 17 00:00:00 2001
Message-Id: <f873c32538f9b920c452354665e8957abaf3d63a.1387389991.git.minovotn@redhat.com>
From: Michal Novotny <minovotn@redhat.com>
Date: Wed, 18 Dec 2013 19:06:06 +0100
Subject: [PATCH] Revert "HMP: Disable drive_add for Red Hat Enterprise Linux"

This reverts commit 8bc60f9c097965e17f1cfa32e66461aacdb11083 because
it caused make check fail.

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 hmp-commands.hx | 2 --
 1 file changed, 2 deletions(-)

diff --git a/hmp-commands.hx b/hmp-commands.hx
index 32cc33c..bfb565b 100644
--- a/hmp-commands.hx
+++ b/hmp-commands.hx
@@ -1072,7 +1072,6 @@ using the specified target.
 ETEXI
 #endif
 
-#if 0 /* Disabled for Red Hat Enterprise Linux */
     {
         .name       = "drive_add",
         .args_type  = "pci_addr:s,opts:s",
@@ -1085,7 +1084,6 @@ ETEXI
         .help       = "add drive to PCI storage controller",
         .mhandler.cmd = drive_hot_add,
     },
-#endif
 
 STEXI
 @item drive_add
-- 
1.7.11.7

